io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+build-option:
+  - append-path: /usr/lib/sdk/rust-stable/bin
+    build-args:
+      - --share=network
```

Comment:
this won't work. there is no network access during the build phase.

you need all the crates downloaded first. There is a tool in https://github.com/flatpak/flatpak-builder-tools to generate a manifest to include.